polkadot-parachain/src/chain_spec/bridge_hubs.rs

```diff
@@ -327,6 +327,10 @@ pub mod rococo {
 						get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
 						get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
 						get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+						// Ethereum Bridge Relayer Accounts
+						get_account_id_from_seed::<sr25519::Public>("Relay"),
```

Comment:
Should only need 2 of these 3 accounts. I think one of them's obsolete. Can you check?

Reply:
`Relay` is not needed. There is an old script that references it. Specifically the `SubBeefy` command in `snowbridge/relayer/magefile.go`.
Removed in https://github.com/Snowfork/snowbridge/pull/785#:~:text=across%20the%20bridge.-,removed%20sub%20beefy%20magefile%20command,-8104ecc